clients/java/src/main/java/io/camunda/client/CamundaClient.java

```diff
@@ -3123,4 +3124,29 @@ IncidentsByElementInstanceSearchRequest newIncidentsByElementInstanceSearchReque
    * @return a builder for the command
    */
   EvaluateExpressionCommandStep1 newEvaluateExpressionCommand();
+
+  /**
+   * Executes a statistics request that returns counts of active process instances with active
+   * incidents, grouped by process definition.
+   *
+   * <p>The statistics are scoped to a specific incident error and therefore require an error hash
+   * code to be provided via the request filter.
+   *
+   * <p>Each result item represents a single process definition (including version and tenant) and
+   * contains the number of active process instances that currently have an incident with the given
+   * error hash code.
+   *
+   * <pre>
+   * camundaClient
+   *  .newIncidentProcessInstanceStatisticsByDefinitionRequest()
+   *  .filter(f -> f.errorHashCode(123456789))
```

Comment:
🔧 The error message hash is required at the moment, so the client could also enforce this by either the method signature (containing the hash code parameter) or by the stepped builder pattern (making the hash code the only first option in the interface before the next steps are allowed). The latter is more complex but also more flexible. We might not need this flexibility here, though.